changelog/change_support_autocorrection_on_redundantmatch.md

```diff
@@ -0,0 +1 @@
+* [#297](https://github.com/rubocop/rubocop-performance/pull/297): Support autocorrection on `Performance/RedundantMatch` when receiver is a Regexp. ([@r7kamura][])
```

Comment:
```suggestion
* [#297](https://github.com/rubocop/rubocop-performance/pull/297): Support autocorrection on `Performance/RedundantMatch` when receiver is a Regexp literal. ([@r7kamura][])
```

Reply:
👍 

I'll update the PR title and the commit message later 👌